modules/utils/package.json

```
@@ -17,10 +17,9 @@
   "types": "lib/index.d.ts",
   "scripts": {
     "build": "upbin tsc --declaration",
-    "ci-test": "upbin nyc npm test --color always",
     "clean": "upbin rimraf $(cat ../../.temporary-files)",
     "lint": "upbin eslint 'src/**/*.ts' 'test/**/*.ts' --fix",
-    "test": "upbin mocha --require ts-node/register 'test/**/*.test.ts' --color always",
+    "test": "upbin nyc mocha --require ts-node/register 'test/**/*.test.ts' --color always",
```

may need upbin for both nyc and mocha
```suggestion
    "test": "upbin nyc upbin mocha --require ts-node/register 'test/**/*.test.ts' --color always",
```

---

also, please name the script `ci-test` for now.